app/src/main/java/com/example/androidtrainingproject/ui/shared/WideButton.kt

```diff
@@ -12,9 +12,9 @@ import com.example.androidtrainingproject.R
 import com.example.androidtrainingproject.ui.theme.Purple
 
 @Composable
-fun WideButton(text: String, enabled: Boolean, onClick: () -> Unit) {
+fun WideButton(modifier: Modifier = Modifier, text: String, enabled: Boolean, onClick: () -> Unit) {
     Button(
-        modifier = Modifier
+        modifier = modifier
             .fillMaxWidth()
             .padding(horizontal = dimensionResource(id = R.dimen.padding_sm_plus))
```

Comment:
If those two are outer paddings (as it looks like), always supply those from outside. Never have outer paddings in a reusable composable as it will impact the whole reusability aspect.

You've done the default chaining here but if you need to supply the outer modifier after the inner one, you can achieve it like this:

```
modifier = Modifier
  .fillMaxWidth()
  .then(modifier)
```